identity-example/build.gradle

```diff
@@ -1,7 +1,7 @@
 plugins {
     id 'com.android.application'
     id 'kotlin-android'
-    id 'org.jetbrains.kotlin.plugin.serialization' version '1.6.10'
+    id 'org.jetbrains.kotlin.plugin.serialization'
```

Comment:
added the specific version to the classpath to avoid replicating it on every declaration.